File: 백준/Bronze/2566. 최댓값/최댓값.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').toString().trim().split('\n').map(x => x.split(' ').map(Number));
let max = 0;
let row, col;

for (let i = 0; i < 9; i++) {
    for (let j = 0; j < 9; j++) {
        const val = input[i][j];
        
        if (val >= max) {
            max = val;
            [row, col] = [i + 1, j + 1];
        }
    }
}

console.log(`${max}\n${row} ${col}`);

File: 백준/Bronze/2566. 최댓값/최댓값.py
max = -1

for i in range(0, 9):
    row = list(map(int, input().split()))
    
    for j in range(0, 9):
        if (row[j] >= max):
            r = i + 1
            c = j + 1
            max = row[j]
            
print(max)
print(r, c)